main/mainTemplate.json

```diff
@@ -28,6 +28,13 @@
       },
       "defaultValue": ""
     },
+    "httpproxy": {
+      "type": "string",
+      "metadata": {
+        "description": "httpproxy."
+      },
+      "defaultValue": ""
```

Comment:
Please remove defaultValue field.

Reply:
We need to keep default in main template otherwise field becomes compulsory.